src/sentry/utils/sdk_crashes/sdk_crash_detection.py

```diff
@@ -64,6 +70,11 @@ def detect_sdk_crash(self, event: Event, event_project_id: int) -> Optional[Even
 
         return None
 
+    def _is_sampled(self, sample_rate: float) -> bool:
+        if sample_rate == 0.0:
+            return False
+        return random.random() <= sample_rate
```

Comment:
[`random.random`](https://python.readthedocs.io/en/stable/library/random.html#random.random) returns a number in [0.0, 1.0), so we can simplify it a bit:
```suggestion
        return random.random() < sample_rate
```
This method may not even be needed after this update.